third_party/move/tools/move-coverage/src/bin/coverage-summaries.rs

```diff
@@ -24,57 +25,92 @@ use std::{
     version
 )]
 struct Args {
-    /// The path to the coverage map or trace file
-    #[clap(long = "input-trace-path", short = 't')]
+    /// The path to the coverage map file
+    #[clap(long = "input-trace-path")]
```

Comment:
Just nit: are we going to remove all abbreviation for all options in the CLI?

Reply:
I removed them for all the files touched here, as it is recommended practice for CLI (see https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/CONTRIBUTING.md?plain=1#L155).